seqr/views/apis/report_api.py

```diff
@@ -510,23 +513,37 @@ def _get_gregor_airtable_data(participants, user):
     )
 
     airtable_metadata_by_participant = {r[PARTICIPANT_ID_FIELD]: r for r in airtable_metadata.values()}
+    rna_metadata_by_smid_record = {}
     for data_type in GREGOR_DATA_TYPES:
         for r in airtable_metadata_by_participant.values():
             data_type_fields = [f for f in r if f.endswith(f'_{data_type}')]
             if data_type_fields:
-                r[data_type.upper()] = {f.replace(f'_{data_type}', ''): r.pop(f) for f in data_type_fields}
+                data_type_metadata = {f.replace(f'_{data_type}', ''): r.pop(f) for f in data_type_fields}
+                r[data_type.upper()] = data_type_metadata
+                if data_type == 'rna':
+                    smid_record_id = data_type_metadata[SMID_FIELD][0]
```

Comment:
SMIDs for RNA are now a foreign reference to the Sample record in airtable, they are no longer strings